website/content/docs/release-notes/nomad/v1_8_x.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: 1.8.x
+description: >-
+  Nomad release notes for version 1.8.x
+---
+
+# Nomad 1.8.0
+
+We are pleased to announce the following Nomad updates.
+
+## Release highlights
+
+- **Long Term Support Release (Enterprise):** Nomad 1.8.0 now introduces a long term supported release which provides extended maintenance (two years of critical fixes provided through minor releases and efficient upgrades (support for direct upgrades from one LTS release to the next). 
```

Comment:
We hard-wrap all our docs.

Reply:
Done, but will add more content for the rest of the release.